content/en/docs/releases/supported-releases/index.md

```diff
@@ -79,7 +79,7 @@ Please keep up-to-date and use a supported version.
 
 | Minor Releases   | Patched versions with no known CVEs           |
 |------------------|-----------------------------------------------|
+| 1.16.x           | 1.16.0+                                       |
 | 1.15.x           | 1.15.3+                                       |
 | 1.14.x           | 1.14.5+                                       |
-| 1.13.x           | 1.13.9+                                       |
-| 1.12 and earlier | None, all versions have known vulnerabilities |
+| 1.13 and earlier | None, all versions have known vulnerabilities |
```

Comment:
Technically 1.13.9 doesn't have a known vulnerability, at least I don't see where we had a fix shown in https://istio.io/latest/news/ which called out 1.13.9. But 1.13 is EOL. We could call out releases with no CVEs that have reached EOL, 1.14 will be there soon, with some different text, such as:

```suggestion
| 1.13                    | End of life. A new CVE will NOT be patched.  |
| 1.12 and earlier | None, all versions have known vulnerabilities |
```

Reply:
Any feedback on if the second suggestion is better or worse would be helpful. I can make the change if needed.

Reply:
Second suggestion looks better to me.

Reply:
Updated adding a separate 1.13 line.